### ISSUE TITLE
Night Owl 2.W (WNVR201 Series)

### ISSUE THREAD

**Issue by feram18** [Sunoo/homebridge-camera-ffmpeg#1210]
**Manufacturer/Model:**
Night Owl 2.W (WNVR201 Series)

**Homebridge Config:**

```json
{
    "name": "Camera FFmpeg",
    "cameras": [
        {
            "name": "Front Door",
            "manufacturer": "NightOwl SP",
            "unbridge": true,
            "videoConfig": {
                "source": "-i http://[IP]:80/cgi-bin/sp.cgi?chn=[CHANNEL]&u=[USER]&p=[PASSWORD]",
                "stillImageSource": "-i http://[USER]:[PASSWORD]@[IP]/cgi-bin/snapshot.cgi?chn=[CHANNEL]&u=[USER]&p=[PASSWORD]",
                "maxStreams": 2,
                "maxWidth": 1920,
                "maxHeight": 1080,
                "maxFPS": 0,
                "audio": false
            }
        }
    ],
    "platform": "Camera-ffmpeg"
}
```

**Additional Information:**
You can try setting the video codec to `h264_omx` but I've found the default `libx264` to be more reliable. Audio is not supported.

**Comment by Relkci** [comment]
Well, I went a loooooong way around this to break into the walled-garden WIFI network that the cameras live on.  I was never able to pull video or stills from the NVR.  Glad to see you had more luck than I did.  Anyway-- My findings, including the ONVIF discovery of the bundled IP cameras (WNIP2 series) living inside the isolated/walled-garden WIFI network can be found here: https://github.com/Relkci/NightOwl-WifiCamera-Config-wnip2
